imports/plugins/included/marketplace/client/containers/marketplaceShops.js

```diff
@@ -9,17 +9,22 @@ const onWorkflowChange = (shopId, value) => {
 };
 
 const composer = (props, onData) => {
-  // Get all shops, excluding the primary shop
-  const shops = Shops.find({
-    _id: {
-      $nin: [Reaction.getPrimaryShopId()]
-    }
-  }).fetch();
+  // Subscribe to MerchantShops
+  const merchantShopSub = Meteor.subscribe("MerchantShops");
```

Comment:
Can you use `Reaction.Subscriptions.MerchantShops` instead? I believe we get caching benefits when the subscriptions manager is used.
Example:

https://github.com/reactioncommerce/reaction/blob/efe64c6d3b292ce8cd9344bf7daa76db4f4fe1ee/imports/plugins/core/i18n/client/containers/languageDropdown.js#L18-L22

Reply:
@impactmass, I made change as requested. However, I observed that I get the cached value of `Reaction.Subscriptions.MerchantShops` unless I refresh the page. I've pushed the update so you can test it.

#### Steps
1. Do a reaction reset
2. Create a shop
3. Check Marketplace shops table. Observe that no shop is displayed
4. Create another shop and repeat step 3. Observe that no shop is still displayed
5. Refresh page and check marketplace shop table again. Observe that shops are now displayed
6. Create another shop and repeat step 3
7. Observe that table is updated with the new shop

Reply:
I just tested it too. I see the behavior you talked about. I can also see that once the subscription kicks in, then you don't need to refresh again. Any idea why this happens?

You can revert back, so that this doesn;t block your PR. We also have other places where we do Meteor.subscribe, so it's okay.